HomeEdu/Scenes/Main/Account/InfoStudentController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  InfoStudentController.swift
+//  HomeEdu
+//
+//  Created by Nguyen Cuong on 8/19/19.
+//  Copyright © 2019 Nguyen Cuong. All rights reserved.
+//
+
+import UIKit
+import Reusable
+import CoreData
+
+final class InfoStudentController: UIViewController {
+
+    @IBOutlet weak private var infoStudentTable: UITableView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configView()
+    }
+    
+    private func configView() {
+        infoStudentTable.delegate = self
+        infoStudentTable.dataSource = self
+        infoStudentTable.register(cellType: InfoStudentCell.self)
+        GetCoreData.shared.getInfoAccount()
+    }
+}
+
+extension InfoStudentController: UITableViewDataSource, UITableViewDelegate {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 3
+    }
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
+        case 0: return Constant.Information.infoStudentTitle
```

Comment:
nên tạo 1 hằng số cho từng sect ion ví dụ
```suggestion
        case SECTION_INFO: return Constant.Information.infoStudentTitle
```